src/cleed++/Core/Coordinate.hh

```
@@ -0,0 +1,68 @@
+      Coordinate& setCoordinate(const coord_t *position);
+
+      // getters
+      double getMagnitude() const;
+      double getX() const;
+      double getY() const;
+      double getZ() const;
+      const coord_t *get_coord() const;
+
+      // other methods
+      void print(FILE *f = stdout);
+    public:
```

Clang (14.0.6)

[warning] redundant access specifier has the same accessibility as the previous access specifier

(readability-redundant-access-specifiers)